src/main/resources/db/migration/V1_16__intervention.sql

```diff
@@ -0,0 +1,23 @@
+create table intervention (
+    id uuid not null,
+    created_at timestamp with time zone not null,
+    type text not null,
+    title text not null,
```

Comment:
there isn't a corresponding field in the entity class for this column

Reply:
Will add in - a few need matching up (wasn't sure how many to add in and so pushed some wip back) - they are also needed for dto